tests/foreman/ui_airgun/test_host.py

```diff
@@ -510,3 +524,764 @@ def test_positive_export(session):
+        organization=org,
+        location=module_loc,
+        host_parameters_attributes=parameters,
+    ).create()
+    additional_host = entities.Host(organization=org, location=module_loc).create()
+    with session:
+        session.organization.select(org_name=org.name)
+        # Check that the hosts are present
+        for host in [param_host, additional_host]:
+            assert session.host.search(host.name)[0]['Name'] == host.name
+        # Check that search by parameter with '<>' operator returns both hosts
+        values = session.host.search('params.{0} <> {1}'.format(param_name, search_param_value))
+        assert {value['Name'] for value in values} == {param_host.name, additional_host.name}
+
+
+@tier2
+def test_positive_search_with_org_and_loc_context(session):
+    """Perform usual search for host, but organization and location used
+    for host create procedure should have 'All capsules' checkbox selected
+
+    :id: 2ce50df0-2b30-42cc-a40b-0e1f4fde3c6f
+
+    :expectedresults: Search functionality works as expected and correct
+        result is returned
+
+    :BZ: 1405496
+
+    :CaseLevel: Integration
+    """
+    org = entities.Organization().create()
+    loc = entities.Location().create()
+    host = entities.Host(organization=org, location=loc).create()
+    with session:
+        session.organization.update(org.name, {'capsules.all_capsules': True})
+        session.location.update(loc.name, {'capsules.all_capsules': True})
+        session.organization.select(org_name=org.name)
+        session.location.select(loc_name=loc.name)
+        assert session.host.search('name = "{0}"'.format(host.name))[0]['Name'] == host.name
+        assert session.host.search(host.name)[0]['Name'] == host.name
+
+
+@tier2
+def test_positive_search_by_org(session, module_loc):
+    """Search for host by specifying host's organization name
+
+    :id: a3bb5bc5-cb9c-4b56-b383-f3e4d3d4d222
+
+    :customerscenario: true
+
+    :expectedresults: Search functionality works as expected and correct
+        result is returned
+
+    :BZ: 1447958
+
+    :CaseLevel: Integration
+    """
+    host = entities.Host(location=module_loc).create()
+    org = host.organization.read()
+    with session:
+        session.organization.select(org_name=ANY_CONTEXT['org'])
+        assert session.host.search('organization = "{0}"'.format(org.name))[0]['Name'] == host.name
+
+
+@tier2
+def test_positive_validate_inherited_cv_lce(session, module_org, module_loc, module_host_template):
+    """Create a host with hostgroup specified via CLI. Make sure host
+    inherited hostgroup's lifecycle environment, content view and both
+    fields are properly reflected via WebUI.
+
+    :id: c83f6819-2649-4a8b-bb1d-ce93b2243765
+
+    :expectedresults: Host's lifecycle environment and content view match
+        the ones specified in hostgroup.
+
+    :CaseLevel: Integration
+
+    :BZ: 1391656
+    """
+    name = gen_string('alpha').lower()
```

Comment:
same

Reply:
so? I see now that we have different scenario here, but it will be nice if you mention that before closing comment